### PR TITLE
Change enemy spawn

### PR DESCRIPTION
**Pull request by JonathanLiu25**
closes #106 
Only some small changes here:
removed unnecessary variables
changed enemy spawn point (away from fantasy state town)

### CODE REVIEW

File: app/states/spaceState.js [hunk 1]
```diff
@@ -67,8 +65,9 @@ const spaceState = {
     // if (Math.random() * 1000 <= 1) this.spawnLoot()
 
     playerObject.movePlayer()
-    itemCollision(playerObject, projectile, localState.loot)
-    enemyCollision(playerObject, projectile, graveyard, localState.loot)
+
+    itemCollision(playerObject, localState.loot)
+    enemyCollision(playerObject, graveyard)
```
Comment (abc516) [inline app/states/spaceState.js]: please add localState.loot again to be the last argument for enemyCollision. Without it, enemyCollision causes an error if the enemy dies.

File: app/states/utils/enemyCollision.js [hunk 2]
```diff
@@ -3,9 +3,12 @@
...
 
-const enemyCollision = (playerObject, projectile, graveyard, lootState) => {
+
+const enemyCollision = (playerObject, graveyard) => {
   Object.keys(GameEnemies).forEach(enemyKey => {
     const enemy = GameEnemies[enemyKey]
+    const projectile = playerObject.getProjectile()
+
     StackQuest.game.physics.arcade.overlap(projectile.bullets, enemy, (target, bullet) => {
       const didDie = enemy.takeDamage(projectile.damage)
```
Comment (abc516) [inline app/states/utils/enemyCollision.js:12]: This function will fail on line 18 when enemy dies.

**Review by abc516 — CHANGES_REQUESTED** [review]
Please address changes noted in comments, then I'll approve.

**Review by gabrielwr — APPROVED** [review]
nice job

**Pull request merged**